packages/firestore/test/unit/specs/limbo_spec.test.ts

```diff
@@ -915,6 +922,66 @@ describeSpec('Limbo Documents:', [], () => {
     }
   );
 
+  specTest(
+    'Limbo resolution throttling with bloom filter application',
```

Comment:
This test doesn't actually trigger "throttling". In order for throttling to be triggered, there has to be more documents in limbo than the value specified to `withMaxConcurrentLimboResolutions()` (which is 2 for this test). So in order to trigger throttling, watch has to put 3 or more documents into limbo. Please update this test case to actually trigger throttling.